app/javascript/courses/review/components/CoursesReview__ChecklistShow.res

```diff
@@ -191,7 +191,7 @@ let make = (
     </div>
     <div
       className="flex justify-end bg-white md:bg-gray-100 border-t sticky bottom-0 px-4 md:px-6 py-2 md:py-4 mt-4">
-      {switch SubmissionDetails.reviewer(submissionDetails) {
+      {switch OverlaySubmission.evaluatorName(overlaySubmission) {
```

Comment:
`evaluatedAt` is a reliable option here, as the coach could be deleted from the school. https://github.com/pupilfirst/pupilfirst/blob/d60c2478ea93cc46837be88c2f0010a224a524f9/app/javascript/courses/review/components/CoursesReview__Editor.res#L685

Reply:
@mahesh-krishnakumar - I noticed that we setting the `reviewer` to `nil` once the submission grading is complete & `evaluatedAt` not available before grading, Updated to check for these both conditions in this commit https://github.com/pupilfirst/pupilfirst/commit/9d6455ee2c69611c2a94790ff971293e28e09d88